tox.ini

```diff
@@ -4,8 +4,9 @@
 [tox]
 no_package = True
 skip_missing_interpreters = True
-env_list = format, lint, static, unit
+env_list = fmt, lint, static, unit, coverage-report
 min_version = 4.0.0
+max-line-length=120
```

Comment:
**suggestion (development):** Adding `dev-requirements.txt` for setting tox version could help setting local environment for contribution.

I got some errors when running `tox devenv -e integration` command. The problem was my tox version. Having `dev-requirements.txt` helps us to have the same tox version.